src/hooks/ParameterConstraintsHooks.js

```diff
@@ -12,6 +12,12 @@ import { useTranslation } from 'react-i18next';
 
 export const useParameterConstraint = (parameterData) => {
   const { t } = useTranslation();
+  const getTranslationLabel = (translationString, fallbackString) => {
+    return t(translationString, fallbackString, {
+      field: t(TranslationUtils.getParameterTranslationKey(parameterData?.id)),
+      constraint: t(TranslationUtils.getParameterTranslationKey(constraint?.id)),
```

Comment:
This line mentions the `constraint` variable that is not declared yet, could we move the function definition down after the variable definition to help readability? (or move the variable declaration up :slightly_smiling_face: )

Reply:
Nice catch, thanks :+1: